server/remote_asset/fetch_server/fetch_server.go

```diff
@@ -31,8 +31,10 @@ import (
 )
 
 const (
-	ChecksumQualifier = "checksum.sri"
-	maxHTTPTimeout    = 60 * time.Minute
+	ChecksumQualifier              = "checksum.sri"
+	QualifierBazelCanonicalID      = "bazel.canonical_id"
+	QualifierBazelHttpHeaderPrefix = "http_header:"
```

Comment:
nit, for consistency w/ the existing code, put the `Qualifier` towards the end of these names? (or rename `ChecksumQualifier` => `QualifierChecksum` but imo that doesn't read as nicely)

```suggestion
	ChecksumQualifier              = "checksum.sri"
	BazelCanonicalIDQualifier      = "bazel.canonical_id"
	BazelHttpHeaderQualifierPrefix = "http_header:"
```